araproc/analysis/hilbert_envelope_snr.py

```diff
@@ -0,0 +1,110 @@
+# this calculates Hilbert envelope SNR for each channel and their average
```

Comment:
sorry this is so nitpicky. can we change the file name to just hilbert.py. i am liking the idea of it providing the utility to calculate the hilbert envelope so others can call it.